src/ERC1271.sol

```diff
@@ -51,8 +51,6 @@ abstract contract ERC1271 {
         (name, version) = _domainNameAndVersion();
         chainId = block.chainid;
         verifyingContract = address(this);
-        salt = salt; // `bytes32(0)`.
-        extensions = extensions; // `new uint256[](0)`.
```

Comment:
This currently causes the compiler to throw warnings that `salt` and `extensions` aren't ever used. In lieu of explicitly assigning a value (which adds gas), we'll live with the warnings for now and address when we more broadly adopt a linter/formatter.

Reply:
After talking with @wilsoncusack decided to revert this change.